devel/libunwind/Portfile

```diff
@@ -49,14 +49,7 @@ destroot.dir    ${build.dir}
 if {${subport} eq "${name}-headers"} {
     revision 0
 
-    supported_archs noarch
-
-    build {}
-
-    destroot.target installhdrs
-    destroot.args \
-        PREFIX="${prefix}"
-    
+    # please keep this test *before* `supported_archs noarch`, which clears `configure.cxx_stdlib`
     if {${os.platform} eq "darwin" && ${os.major} < 11 && ${configure.cxx_stdlib} eq "libc++"} {
```

Comment:
An alternate fix might be to change this to check `cxx_stdlib` rather than `configure.cxx_stdlib`, since it's about how other ports will be building. But the way it is also works.